libraries/compile-examples/entrypoint.sh

```diff
@@ -41,5 +51,5 @@ ln -s $PWD $HOME/Arduino/libraries/.
 EXAMPLES=`find examples/ -name '*.ino' | xargs dirname | uniq`
 for EXAMPLE in $EXAMPLES; do
   echo Building example $EXAMPLE
-  arduino-cli compile --verbose --warnings all --fqbn $FQBN $EXAMPLE
+  arduino-cli compile --verbose --warnings all --fqbn $BOARD $EXAMPLE
```

Comment:
I'd prefer to keep the `$FQDN` variable name here as board is a "fully qualified board name", so would suggest to change the `FQBN=$2` line to something like `FQBN_ARG=$2`.

Reply:
OK, I have done as requested. I also wasn't happy with that `BOARD` variable name. It's a bit awkward that the action's `fqbn` argument is now used for more than just the FQBN, but that was the only solution I could find that wouldn't cause breakage.